second_try.js

```diff
@@ -0,0 +1,61 @@
+var searchTerm;
+var currentBackgroundNumber = 1;
+var currentBackground;
+var transitionTime = 0.5;
+var slideshowInterval;
```

Comment:
This is an excellent solution! In the future, I would recommend reducing the amount of global variables, and maybe creating an options object that the user can edit if they want to change things like the transition time or slide interval